app/assets/stylesheets/regions/mastheads/boxes-masthead.scss

```diff
@@ -227,47 +227,4 @@
     min-height: 340px !important;
     padding-bottom: 0;
   }
-}
-
```

Comment:
No, the code that is showing as deleted was moved `app/assets/stylesheets/regions/slider-masthead-old.scss`